### ISSUE TITLE
Interactive window with Matplotlib and notebook option

### ISSUE THREAD

**Issue by PierreMarchand20** [microsoft/vscode-jupyter#3691]
## Environment data

- VS Code version: 1.30.1
- Extension version (available under the Extensions sidebar): XXX
- OS and version: macOS Mojave 
- Python version (& distribution if applicable, e.g. Anaconda): 3.7.2
- Type of virtual environment used (N/A | venv | virtualenv | conda | ...): N/A
- Relevant/affected Python packages and their versions: Matplotlib

Is `%matplotlib notebook` supposed to work in vscode ? In my case, it does not:
```
ReferenceError: IPython is not defined
    at eval (eval at runCodeHere (webpack:///./node_modules/@nteract/transforms/lib/javascript.js?), <anonymous>:767:1)
    at runCodeHere (webpack:///./node_modules/@nteract/transforms/lib/javascript.js?:32:12)
    at JavaScript.componentDidMount (webpack:///./node_modules/@nteract/transforms/lib/javascript.js?:60:7)
    at rh (webpack:///./node_modules/react-dom/cjs/react-dom.production.min.js?:213:482)
    at Xg (webpack:///./node_modules/react-dom/cjs/react-dom.production.min.js?:205:159)
    at Yg (webpack:///./node_modules/react-dom/cjs/react-dom.production.min.js?:204:174)
    at If (webpack:///./node_modules/react-dom/cjs/react-dom.production.min.js?:200:307)
    at Object.enqueueForceUpdate (webpack:///./node_modules/react-dom/cjs/react-dom.production.min.js?:131:37)
    at MainPanel.E.forceUpdate (webpack:///./node_modules/react/cjs/react.production.min.js?:13:311)
    at MainPanel._super.MainPanel._this.updateOrAdd (webpack:///./src/datascience-ui/history-react/MainPanel.tsx?:341:23)
```

**Comment by franva** [comment]
how about now?

Does VSCode support jupyter extensions?

**Comment by DonJayamanne** [comment]
`%matplotlib widget` works today. 
Please try that.

**Comment by franva** [comment]
thanks @DonJayamanne 
just tried and it is almost perfect with 1 glitch.

The animation doesn't start by itself(although the sensory data has been streaming through), 
it only starts animating once I resized its plot by dragging a small triangle at the right bottom corner.

I have uploaded a screen recording for this glitch.

https://user-images.githubusercontent.com/5151146/116352435-7cc74080-a838-11eb-83bd-9fe931b6bc77.mp4
